templates/Vagrantfile.erb

```diff
@@ -124,6 +124,12 @@ Vagrant.configure("2") do |c|
     <% else  %>
     p.<%= key %> = <%= value%>
     <% end  %>
+  <% when "cloudstack" %>
```

Comment:
Because this is the exact same contents as openstack, can you use `<% when "openstack", "cloudstack" %>` instead so we don't duplicate so much?